src/components/Header.vue

```diff
@@ -22,7 +22,7 @@
                             <font-awesome-icon :icon="['fab', 'github']" />About
                         </b-nav-item>
                         <li class="nav-item">
-                            <b-button href="#" variant="dark" class="font-weight-bold">Nominate</b-button>
+                            <b-button href="/nominate" variant="dark" class="font-weight-bold">Nominate</b-button>
```

Comment:
You should use a [router-link](https://router.vuejs.org/api/#router-link) instead, this is way you get a 404 on netlify, because when you use a html link it will reload the page.